src/AlgAss/Types.jl

```diff
@@ -226,8 +272,22 @@ mutable struct GroupAlgebraElem{T, S} <: AbstractAssociativeAlgebraElem{T}
     z.coeffs = coeffs
     return z
   end
+
+  function GroupAlgebraElem{T, S}(A::S, coeffs::SRow{T}) where {T, S}
+    z = new{T, S}()
+    z.parent = A
+    z.coeffs_sparse = coeffs
+    return z
+  end
 end
 
+__elem_index(A, g) = get!(A.group_to_base, g) do
+        A.ind += 1
+        resize!(A.base_to_group, max(A.ind, length(A.base_to_group)))
+        A.base_to_group[A.ind] = g
+        return A.ind
```

Comment:
So when would `length(A.base_to_group)` ever exceed `A.ind`? Right now I am wondering why you even need `A.ind` and can't just do this:
```suggestion
        push!(A.base_to_group, g)
        return length(A.base_to_group)
```

Reply:
Yes, I think I wanted to do work with A.base_to_group thing differently, but it does not make any sense now. Will do as you suggest.